src/features/device-config/device-config.ts

```diff
@@ -67,6 +67,11 @@ export const generateConfig = async (
 			req.query.provisioningKeyDescription ??
 			'Automatically generated for an image download or config file generation';
 
+		apiKeyOptions.expiryDate =
+			req.body.provisioningKeyExpiry ??
+			req.query.provisioningKeyExpiry ??
```

Comment:
NIT: How about v for consistency?
```suggestion
			req.body.provisioningKeyExpiryDate ??
			req.query.provisioningKeyExpiryDate ??
```